src/wranglerAction.ts

```diff
@@ -379,46 +475,23 @@ async function wranglerCommands(
 			setOutput("command-output", stdOut);
 			setOutput("command-stderr", stdErr);
 
-			// Check if this command is a workers deployment
-			if (command.startsWith("deploy") || command.startsWith("publish")) {
-				const { deploymentUrl } = extractDeploymentUrlsFromStdout(stdOut);
-				setOutput("deployment-url", deploymentUrl);
-			}
-			// Check if this command is a pages deployment
-			if (
-				command.startsWith("pages publish") ||
-				command.startsWith("pages deploy")
-			) {
-				const pagesArtifactFields = await getDetailedPagesDeployOutput(
-					config.WRANGLER_OUTPUT_DIR,
-				);
-
-				if (pagesArtifactFields) {
-					setOutput("deployment-url", pagesArtifactFields.url);
-					// DEPRECATED: deployment-alias-url in favour of pages-deployment-alias, drop in next wrangler-action major version change
-					setOutput("deployment-alias-url", pagesArtifactFields.alias);
-					setOutput("pages-deployment-alias-url", pagesArtifactFields.alias);
-					setOutput("pages-deployment-id", pagesArtifactFields.deployment_id);
-					setOutput("pages-environment", pagesArtifactFields.environment);
-					// Create github deployment, if GITHUB_TOKEN is present in config
-					await createGitHubDeploymentAndJobSummary(
-						config,
-						pagesArtifactFields,
-					);
-				} else {
-					info(
-						config,
-						"Unable to find a WRANGLER_OUTPUT_DIR, environment and id fields will be unavailable for output. Have you updated wrangler to version >=3.81.0?",
-					);
-					// DEPRECATED: deployment-alias-url in favour of pages-deployment-alias, drop in next wrangler-action major version change
-					const { deploymentUrl, aliasUrl } =
-						extractDeploymentUrlsFromStdout(stdOut);
-
-					setOutput("deployment-url", deploymentUrl);
-					// DEPRECATED: deployment-alias-url in favour of pages-deployment-alias, drop in next wrangler-action major version change
-					setOutput("deployment-alias-url", aliasUrl);
-					setOutput("pages-deployment-alias-url", aliasUrl);
-				}
+			switch (true) {
```

Comment:
Look I kinda like it; But it's kinda clever. It took me a little too long to parse exactly and make sure you were doing what you wanted to do. It would be _much more clear_ with simple `if` statements.